.github/scripts/release-against-rancher.sh

```diff
@@ -28,9 +28,14 @@ if [ ! -e ~/.gitconfig ]; then
     git config --global user.email fleet@suse.de
 fi
 
-sed -i -e "s/ENV CATTLE_FLEET_VERSION=.*$/ENV CATTLE_FLEET_VERSION=${NEW_CHART_VERSION}+up${NEW_FLEET_VERSION}/g" package/Dockerfile
-
-git add package/Dockerfile
+if [ -e build.yaml ]; then
+    sed -i -e "s/fleetVersion: .*$/fleetVersion: ${NEW_CHART_VERSION}+up${NEW_FLEET_VERSION}/" build.yaml
+    git add build.yaml
```

Comment:
As per https://github.com/rancher/rancher/blob/release/v2.8/docs/build.md#update-an-existing-value, editing `build.yaml` is not enough. We also need to regenerate the constants file with versions:
```suggestion
    go generate
    git add build.yaml pkg/buildconfig/constants.go
```